lib/media/track/remotemediatrack.js

```diff
@@ -4,75 +4,104 @@ function mixinRemoteMediaTrack(AudioOrVideoTrack) {
   /**
    * A {@link RemoteMediaTrack} represents a {@link MediaTrack} published to a
    * {@link Room} by a {@link RemoteParticipant}.
-   * @property {boolean} isSubscribed - Whether the {@link RemoteMediaTrack} is
-   *   subscribed by the {@link LocalParticipant}
-   * @property {Track.SID} sid - The SID assigned to the {@link RemoteMediaTrack}
-   * @emits RemoteTrack#unsubscribed
+   * @emits RemoteMediaTrack#disabled
+   * @emits RemoteMediaTrack#enabled
+   * @emits RemoteMediaTrack#unsubscribed
    */
   return class RemoteMediaTrack extends AudioOrVideoTrack {
```

Comment:
In the class hierarchy, this is basically at the level of RemoteDataTrack, even though it's concrete implementations are RemoteAudioTrack and RemoteVideoTrack?

Reply:
@markandrus Yes.